### PR TITLE
[bitnami/external-dns] Add secretAnnotations value

### PR DESCRIPTION
**Pull request by AntonioAlfrz**
**Description of the change**
This PR adds the value secretAnnotations to allow to add annotations to the secret

**Benefits**

- More customization
- Be able to integrate it with other tools like [SealedSecrets](https://github.com/bitnami-labs/sealed-secrets#managing-existing-secrets)

**Possible drawbacks**

**Applicable issues**

**Additional information**

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [X] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/external-dns/README.md [hunk 1]
```diff
@@ -187,6 +187,7 @@ The following table lists the configurable parameters of the external-dns chart
 | `podSecurityContext.fsGroup`           | Group ID for the container                                                                                                                                                                                      | `1001`                                                  |
 | `podSecurityContext.runAsUser`         | User ID for the container                                                                                                                                                                                       | `1001`                                                  |
 | `priorityClassName`                    | priorityClassName                                                                                                                                                                                               | `""`                                                    |
+| `secretAnnotations`                    | Additional annotations to apply to the secret | `{}`
```
Comment (miguelaeh) [inline bitnami/external-dns/README.md:190]: Could you please add the same format as the rest of the rows?

File: bitnami/external-dns/templates/secret.yaml [hunk 2]
```diff
@@ -4,6 +4,9 @@ kind: Secret
 metadata:
   name: {{ template "external-dns.fullname" . }}
   labels: {{ include "external-dns.labels" . | nindent 4 }}
+  {{- if .Values.secretAnnotations }}
+  annotations: {{ toYaml .Values.secretAnnotations | nindent 4 }}
```
Comment (miguelaeh) [inline bitnami/external-dns/templates/secret.yaml:8]:
```suggestion
  annotations: {{- include "common.tplvalues.render" (dict "value" .Values.secretAnnotations "context" $) | nindent 4 }}
```

**Review by miguelaeh — CHANGES_REQUESTED** [review]
Thank you very much for the PR!
Please take a look at my comments

**Review by dani8art — APPROVED** [review]
Thanks for your PR, LGTM!

**Pull request merged**